.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,21 @@
 <!--
-# nf-core/staramr pull request
+# staramr: nextflow pipeline pull request
 
-Many thanks for contributing to nf-core/staramr!
+Many thanks for contributing to staramr: nextflow pipeline!
 
 Please fill in the appropriate checklist below (delete whatever is not relevant).
 These are the most common things requested on pull requests (PRs).
 
 Remember that PRs should be made against the dev branch, unless you're preparing a pipeline release.
 
-Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/staramr/tree/master/.github/CONTRIBUTING.md)
+Learn more about contributing: [CONTRIBUTING.md](https://github.com/phac-nml/nf-core-staramr/tree/main/.github/CONTRIBUTING.md)
```

Comment:
The old repo name will redirect to the new `staramrnf` name, but might as well change this to the new `staramrnf` name here.

Reply:
Good find! I will make sure there are no others.